### PR TITLE
magento/devdocs#3220: Prioritizing Plugins section of Plugins (Interceptors) is not clear

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

Fixes  https://github.com/magento/devdocs/issues/3200.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.2/extension-dev-guide/plugins.html
- https://devdocs.magento.com/guides/v2.3/extension-dev-guide/plugins.html

## Links to Magento source code

- ...

CC: @shrielenee 

Thank you!

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by shrielenee** [comment]
Hey @atwixfirster --- It looks like these changes are just grammatical. I think to clear up https://github.com/magento/devdocs/issues/3200 with this PR, you should add some detail like you did in that issue: https://github.com/magento/devdocs/issues/3200#issuecomment-522369039.

Do you think an explanation like that is appropriate here?

**Comment by atwixfirster** [comment]
> Do

@shrielenee , if my explanation is simple and understandable for you then we can use it :)

Thanks

**Comment by atwixfirster** [comment]
@osrecio, I know you are already approved this PR but could you please take a look at last @shrielenee question. 

Thank you!

**Comment by atwixfirster** [comment]
> Hey there! Thanks for the contribution. Looks like this section needed some improvements. How about improving it a tidbit more? I provided some suggestions.

Hi, @hguthrie!

I can not say NO :)

File: guides/v2.2/extension-dev-guide/plugins.md [hunk 1]
```diff
@@ -278,7 +276,7 @@ Given the following plugins observing the same method with the following propert
 | **after**                | afterDispatch()  | afterDispatch()                | afterDispatch()                |                  |
 | :----------------------: | :--------------: | :----------------------------: | :----------------------------: | :--------------: |
 
-The execution flow will be as follows:
+The execution flow is:
```
[the remark this answers is not in the document]
Comment (atwixfirster) [reply guides/v2.2/extension-dev-guide/plugins.md]: **implemented**

File: guides/v2.2/extension-dev-guide/plugins.md [hunk 2]
```diff
@@ -250,23 +250,21 @@ class MyUtilityUpdater
...
+The `sortOrder` property for plugins determine when their `before`, `around`, or `after` methods get called when more than one plugin is observing the same method.
 
 The prioritization rules for ordering plugins:
 
 * Before the execution of the observed method, Magento will execute plugins from lowest to greatest `sortOrder`.
 
-  * During each plugin execution, Magento executes the current plugin's before method.
-  * After the before plugin completes execution, the current plugin's around method will wrap and execute the next plugin or observed method.
+  * During each plugin execution, Magento executes the current plugin's `before` method.
+  * After the `before` plugin completes execution, the current plugin's `around` method will wrap and execute the next plugin or observed method.
```
[the remark this answers is not in the document]
Comment (atwixfirster) [reply guides/v2.2/extension-dev-guide/plugins.md]: **implemented**

File: guides/v2.2/extension-dev-guide/plugins.md [hunk 3]
```diff
@@ -250,23 +250,21 @@ class MyUtilityUpdater
 
 ### Prioritizing plugins
 
-The `sortOrder` property for plugins determine when their before, after, or around methods get called when more than one plugins are observing the same method.
+The `sortOrder` property for plugins determine when their `before`, `around`, or `after` methods get called when more than one plugin is observing the same method.
```
[the remark this answers is not in the document]
Comment (atwixfirster) [reply guides/v2.2/extension-dev-guide/plugins.md]: **implemented**

**Pull request merged**